lib/node_modules/@stdlib/ndarray/docs/types/index.d.ts

```diff
@@ -28,6 +28,7 @@ import ndarray = require( '@stdlib/ndarray/ctor' );
 import ndarrayDataBuffer = require( '@stdlib/ndarray/data-buffer' );
 import defaults = require( '@stdlib/ndarray/defaults' );
 import dispatch = require( '@stdlib/ndarray/dispatch' );
+import dispatchBy = require( '@stdlib/ndarray/dispatch-by' );
```

Comment:
This was triggered by https://github.com/stdlib-js/stdlib/commit/b1e4cca30147188064b2275a0242a0334befbcd1. I don't believe this function should be added to the namespace declarations.